docs/reference/launcher.md

```diff
@@ -12,6 +12,11 @@ Options:
           [env: MODEL_ID=]
           [default: mistralai/Mistral-7B-Instruct-v0.1]
 
+      --env <HUGGING_FACE_HUB_TOKEN>
```

Comment:
Should be `--env HUGGING_FACE_HUB_TOKEN=<HUGGING_FACE_HUB_TOKEN>`, according to the rest of the documentation.

Reply:
Correct. This is also something that should be set in the container, not on the lorax launcher itself.

I can add some quick docs to reflect this.